pkg/api/secretbootstrap/secretboostrap.go

```diff
@@ -239,3 +239,10 @@ func (c *Config) resolve() error {
 
 	return utilerrors.NewAggregate(errs)
 }
+
+const OSDGlobalPullSecretGroupName = "osd_global_pull_secret"
+
+// OSDGlobalPullSecretGroup returns the list of the OSD cluster names where we need to partially manage the global pull secret
+func (c *Config) OSDGlobalPullSecretGroup() []string {
+	return c.ClusterGroups[OSDGlobalPullSecretGroupName]
+}
```

Comment:
The code in the PR LGTM, but I see the pull secrets in all our clusters have the same entries (key names).  Any reason not to do this for all of them?

Reply:
Good one.
This is more of doing the things we have to. :) 
1. b01, b02: OCP clusters. Our automation manages the global pull secret as a whole.
2. b03-b05: Will be managed after this PR.
3. arm01, vsphere: not managed by us.
4. hive, app.ci: also OSD. But no e2e jobs run there.